sonar-project.properties

```diff
@@ -11,7 +11,7 @@ sonar.sources=.
 
 sonar.exclusions=**/Makefile, **/*.Dockerfile, api/app/data/**
 sonar.test.exclusions=*.feature
-sonar.coverage.exclusions=api/scripts/**, api/alembic/**
+sonar.coverage.exclusions=api/scripts/**, api/alembic/**, api/app/tests/**
```

Comment:
What is this for?

Reply:
We don't need coverage for test code. Then we'd need to write tests for our tests.

Reply:
Oh I see, I thought it will always be 100% lol

Reply:
Well - it's kinda messy. It wants the test code to run, so if there's any kind of conditional that doesn't happen in your test code, that comes up as not covered. So in this case, because there's a conditional on AUTO_MAKE_FIXTURES, there's test code, that just doesn't run.
That's all good an well, yes, it's code that doesn't run as part of a unit test - but I really don't care. I mean, it's nice to know if we have unit tests that don't get run at all - but that's a different problem.

Reply:
So I'm fine with just a blanket: don't code unit tests as part of your coverage.